src/Core/Content/MailTemplate/Api/MailActionController.php

```diff
@@ -27,15 +28,16 @@ class MailActionController extends AbstractController
      */
     public function __construct(
         private readonly AbstractMailService $mailService,
-        private readonly StringTemplateRenderer $templateRenderer
+        private readonly StringTemplateRenderer $templateRenderer,
+        private readonly MailTemplateService $mailTemplateService,
     ) {
     }
 
     #[Route(
         path: '/api/_action/mail-template/send',
         name: 'api.action.mail_template.send',
-        methods: ['POST'],
-        defaults: ['_acl' => ['api_send_email']]
+        defaults: ['_acl' => ['api_send_email']],
+        methods: ['POST']
```

Comment:
Request::METHOD_POST
```suggestion
        methods: [Request::METHOD_POST]
```

Reply:
same for the other ones 👇🏻